justfile

```diff
@@ -0,0 +1,41 @@
+list:
+    just --list
+
+install-typeshare:
```

Comment:
should we have `install SERVICE` ?  to follow just standard

Reply:
Yes, we can add them later, I'm moving existed targets from `Makefile`